e2e/support/server.js

```diff
@@ -0,0 +1,26 @@
+const net = require('net')
```

Comment:
Can this use Typescript?

Reply:
This is not straightforward, should probably be a separate PR.